### ISSUE TITLE
Increase Windows Size? Edited Constant.java didn't seem to scale all things up

### ISSUE THREAD

**Issue by HRNPH** [catabriga/graphwar#11]
I want to play on a bigger screen so I edited a Constant file that seems to contain windows size config and recompile it,
IT WORKED 👍 
but the problem is Everything didn't scale up accordingly to the window size (I'm not a java programmer)
so I think Constant for other UI elements lies in other parts, but I don't know where to look at

**Comment by HRNPH** [comment]
![image](https://user-images.githubusercontent.com/51855316/210410417-135e278c-9449-4f6e-907c-5321870c1f24.png)
![image](https://user-images.githubusercontent.com/51855316/210410454-4fd4746f-c41a-4bd8-a2ee-9caaa31cf815.png)

**Comment by cspablocortez** [comment]
Yeah they won't scale because they're PNG images inside the rsc/ directory.

**Comment by Eloitor** [comment]
```
alias graphwar='wmname LG3D; java -Dsun.java2d.uiScale=2 -jar ~/builds/graphwar/graphwar.jar'
```

**Comment by HRNPH** [comment]
thanks, will try.
